packages/core/src/package-utils/bundle.ts

```diff
@@ -45,6 +45,9 @@ export const createBundle = async (
       inlineDynamicImports: false,
       interop: 'compat',
       format,
+      hoistTransitiveImports: false,
+      // prevent chunks -- we'll manage them ourselves
+      experimentalMinChunkSize: 0,
```

Comment:
This is a bit of a head scratcher. The default value is 1, and [according to the docs][minchu]...

> Set a minimal chunk size target in Byte for code-splitting setups. When this value is set to the default of 1, Rollup will try to merge chunks that do not contain code except imports and reexports into other chunks. A merge will only be performed if it does not change what side effects are executed when any entry is loaded. For the value of 1, only merges are permitted that do no increase the amount of code loaded for any entry.

...the value 0 should have the same effect. The suggested way of opting out — via the value `Infinity` — doesn't have the desired effect for us.

[minchu]: https://rollupjs.org/configuration-options/#output-experimentalminchunksize